standards/API_STANDARDS.md

```diff
@@ -13,26 +13,30 @@ API standards
 
 	  When pagination is required, the fields work as:
 
-	  `limit` - max number of items we're returning in this response (e.g. 20, or 50, rather than all we use atm)
+	  `limit` - max number of items we're returning in this response.
+	  * If value is not set a configurable default limit should be used (e.g. 20, or 50, rather than returning all items).
```

Comment:
Should we recommend a default as I believe some places have a default of 100 which I would argue is too high?

Reply:
Yes 100 seems excessive as a default to me too, 20 seems like a good number to me. Arguably this could be made to be configurable, but might pose a problem if we update the swagger docs with the default values.